de.fhdo.lemma.eclipse.ui/src/de/fhdo/lemma/eclipse/ui/ExtractLemmaModelsFromOpenApiController.xtend

```diff
@@ -0,0 +1,25 @@
+package de.fhdo.lemma.eclipse.ui
+
+import org.eclipse.core.commands.AbstractHandler
+import org.eclipse.core.commands.ExecutionEvent
+import org.eclipse.core.commands.ExecutionException
+import org.eclipse.ui.PlatformUI
+import org.eclipse.jface.dialogs.MessageDialog
+import de.fhdo.lemma.eclipse.ui.specify_url_dialog.commands.SpecifyUrlHandler
+
+/**
+ * Controller for handling the extraction of lemma models from openapi specifications in the UI.
+ *
+ * @author <a href="mailto:jonas.sorgalla@fh-dortmund.de">Jonas Sorgalla</a>
+ */
+class ExtractLemmaModelsFromOpenApiController extends AbstractHandler {
+    val SHELL = PlatformUI.workbench.activeWorkbenchWindow.shell
+    SpecifyUrlHandler specifyUrlHandler
```

Comment:
Why not initialize the variable here directly via `val specifyUrlHandler = new SpecifyUrlHandler()`?